wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/sample/Robot.java

```diff
@@ -91,24 +91,32 @@ public void robotInit() {
 	 */
 	@Override
 	public void autonomous() {
-		String autoSelected = m_chooser.getSelected();
-		// String autoSelected = SmartDashboard.getString("Auto Selector",
```

Comment:
This code is correct per the comments (sendable chooser by default for SD, commented out code for LabVIEW dashboard compatibility.